dali/operators/generic/reduce/reduce.cc

```diff
@@ -29,6 +29,10 @@ DALI_SCHEMA(ReduceBase)
 
 Not providing any axis results in reduction of all elements.)code",
     std::vector<int>{ })
+  .AddOptionalArg("axis_names", R"code(Axis or axes along which reduction is performed, given by name.
```

Comment:
nitpick: Maybe it'd be better to remove the default value from here. It'd look better in the documentation
```suggestion
  .AddOptionalArg<TensorShape>("axis_names", R"code(Axis or axes along which reduction is performed, given by name.
```

Reply:
Done